source/resources/configs/config_default/target_storage/target_storage.cfg

```diff
@@ -34,7 +36,7 @@ BIPARTITE FALSE
 RELEVANCE_THRESHOLD 0.9
 
 #Format of crawled data, either CBOR or PLAINFILE
-DATA_FORMAT FILE
```

Comment:
This trick works when accessing an exposed Docker container.  If Memex Explorer is linked, we'd need to point to a host named elasticsearch instead.